src/iotile/models/properties.ts

```diff
@@ -0,0 +1,8 @@
+export class Properties {
```

Comment:
I am uneasy about having one `Properties` model instead of an array of `Property` records, but other than gut feeling, I don't have a way to justify me worries, so we can leave like this for now